src/lazy/text/buffer.rs

```diff
@@ -776,22 +776,26 @@ impl<'top> TextBufferView<'top> {
         // Scan ahead to find the end of this list.
         let list_body = self.slice_to_end(1);
         let sequence_iter = RawTextListIterator_1_1::new(list_body);
-        let (span, child_exprs) =
-            match TextListSpanFinder_1_1::new(self.allocator, sequence_iter).find_span() {
-                Ok((span, child_exprs)) => (span, child_exprs),
-                // If the complete container isn't available, return an incomplete.
-                Err(IonError::Incomplete(_)) => return Err(nom::Err::Incomplete(Needed::Unknown)),
-                // If invalid syntax was encountered, return a failure to prevent nom from trying
-                // other parser kinds.
-                Err(e) => {
-                    return {
-                        let error = InvalidInputError::new(self)
-                            .with_label("matching a v1.1 list")
-                            .with_description(format!("{}", e));
-                        Err(nom::Err::Failure(IonParseError::Invalid(error)))
-                    }
+        let (span, child_exprs) = match TextListSpanFinder_1_1::new(
+            self.context.allocator(),
```

Comment:
🗺️ The change from `self.allocator` to `self.context.allocator()` caused `rustfmt` to reflow the entire expression.